overview.tex

```diff
@@ -15,10 +15,15 @@ \chapter{System Overview} \label{overview}
 OpenOCD, which may include a hardware driver) to communicate with Debug
 Transport Hardware (eg.  Olimex USB-JTAG adapter).
 The Debug Transport Hardware connects the Debug Host to the Platform's Debug
-Transport Module (DTM).  The DTM provides access to the Debug Module (DM) using
-the Debug Module Interface (DMI).
+Transport Module (DTM).  The DTM provides access to one or more Debug Modules
+(DMs) using the Debug Module Interface (DMI).
 
-The DM allows the debugger to halt any hart in the platform. Abstract commands
+Each hart in the system is controlled by exactly one DM. Harts may be
```

Comment:
system -> platform, since that seems to be the term we are using.

Reply:
Done. Thanks.